scripts/prepare_layer_files.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-pip install wheel
+pip install wheel --quiet
```

Comment:
Did you change that on purpose?

Reply:
yes :)
It's not relevant to that PR, but just doing good in general :)